packages/react-core/src/components/Wizard/examples/Wizard.md

```diff
@@ -859,3 +860,138 @@ class WizardInModal extends React.Component {
   }
 }
 ```
+
+### Wizard with drawer
+
+```js isBeta
+import React from 'react';
+import {
+  Button,
+  Drawer,
+  DrawerActions,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerContentBody,
+  DrawerHead,
+  DrawerPanelContent,
+  DrawerSection,
+  Wizard
+} from '@patternfly/react-core';
+
+class WizardWithDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+      isExpanded: true,
+      sectionGray: false,
+      panelGray: true,
+      contentGray: false
+    };
+    this.handleModalToggle = () => {
+      this.setState(({ isOpen }) => ({
+        isOpen: !isOpen
+      }));
+    };
+
+    this.drawerRef = React.createRef();
+
+    this.onExpand = () => {
+      this.drawerRef.current && this.drawerRef.current.focus();
+    };
+
+    this.onClick = () => {
+      console.log('test');
+      const isExpanded = !this.state.isExpanded;
+      this.setState({
+        isExpanded
+      });
+    };
+
+    this.onCloseClick = () => {
+      this.setState({
+        isExpanded: false
+      });
+    };
+  }
+
+  render() {
+    const { isOpen, isExpanded } = this.state;
+
+    const panel1Content = (
+      <DrawerPanelContent widths={{ default: 'width_25' }} colorVariant={DrawerColorVariant.light200}>
```

Comment:
Thank you @nicolethoen @mcarrano for your thorough reviews and helpful feedback! 

The width of the drawer panel is set through the `widths` prop of `DrawerPanelContent`, which a consumer can customize as shown in the demo here.

I've set the smallest available width as the `default` value, and the demo has been updated to reflect this!